### DISCUSSION TITLE
E-booki do rozdania!

### DISCUSSION THREAD

**Discussion by kamil-mrzyglod** [AzureCommunityPL/group#36] [Community Content]
Hej, poszukuję tak 15-20 osób na tzw. entry level(czyli raczej zaczynających przygodę z Azure bądź całkowicie zielonych), którzy są zainteresowni otrzymaniem bezpłatnego egzemplarza książki "Learn Azure Administration". Ponieważ wiadomo, że nie ma nic za darmo, w zamian za ebooka prosiłbym się o odwdzięczenie się w formie recenzji - nawet kilka zdań wrzuconych na Amazona :)

Jeśli jesteście zainteresowani, zarejestrujcie się tutaj - https://authors.packtpub.com/reviewer-form/ podając **978-1-83855-145-2** jako reference number.

Dzięki!

**Comment by mirekszym** [comment]
proszę bardzo :)

**Reply by kamil-mrzyglod** [reply]
Udało się zarejestrować?

**Comment by senionit** [comment]
Chętny zarówno do czytania jak i recenzji.

**Reply by kamil-mrzyglod** [reply]
To tak jak pisałem - musisz się zarejestrować pod podanym linkiem i podać reference / referral number aby potem dostać dostęp do ebooka.

**Reply by senionit** [reply]
Udało się zarejestrować z podanym numerem. Czy teraz muszę czekać na aktywację ebooka?

**Reply by kamil-mrzyglod** [reply]
Tak, wydawnictwo będzie wysyłać ebooka do wszystkich zarejestrowanych :)

**Comment by kamil-mrzyglod** [comment]
Podbijam temat bo jeszcze są darmowe egzemplarze do wzięcia :)

**Reply by ostrowto** [reply]
Zarejestrowano

**Comment by kamil-mrzyglod** [comment]
Podbitka - prośba do osób, które zapisały się na e-booka na wysłanie maila na kamil@thecloudtheory.com - mam dla Was info do przekazania :)
